src/com/elixr/training/Main.java

```diff
@@ -0,0 +1,72 @@
+package com.elixr.training;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.StringTokenizer;
+import java.lang.String;
+
+public class Main {
+
+    public static void main(String[] args) {
+
+                if(args.length!=2)
+                {
+                    System.out.println("enter the valid path");
```

Comment:
The message could appear also because search word is missing right? Update the message accordingly. Also start all sysout messages with capital letters.

Reply:
all above changes are done